migration-guides/v6.0.0.md

```diff
@@ -9,6 +9,7 @@ This version contains the following breaking changes:
 1. `<CloudinaryFileInput />` and `<FileInput />` components now store an array of objects representing the file data as opposed to a string
 1. `<CloudinaryFileInput />` and `<FileInput />` components no longer accept an `onLoad` prop
 1. The `previewComponent` for a file input no longer receives a `value` prop and `file` is a file object (with the url)
+1. The tag on `<Spinner />` now uses the class `spinner` in place of an id which can be custom
```

Comment:
"...which can be custom" implies to me that the spinner class can be overridden. Our implementation allows for appending _additional_ classes, but not customizing the base class. Would you mind updating this language?